src/components/HeatMap/HeatMap.js

```diff
@@ -1,16 +1,71 @@
-import React from 'react';
-import { Link } from 'react-router-dom';
-import table from './table.png';
-import config from '../../config';
-import { Wrapper, Image } from './HeatMap.styles';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Row from './Row';
+import Header from './Header';
+import Footer from './Footer';
+import { Wrapper } from './HeatMap.styles';
 
 
-export default function HeatMap() {
+const dayLabels = [
+  'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday',
+];
+
+export default function HeatMap({ data }) {
+  const [tableData, setTableData] = useState([]);
+  const [activeCell, setActiveCell] = useState([]);
+  const [timezone, setTimezone] = useState('');
+  const [timezoneDifference, setTimezoneDifference] = useState(0);
+
+
+  useEffect(() => {
+    const clientTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
I don't think you need to store that in a state variable since it doesn't change. You can simply use `Intl.DateTimeFormat().resolvedOptions().timeZone` in the footer directly